back/sssscs/src/main/resources/application.properties

```diff
@@ -1 +1,12 @@
+spring.datasource.url=jdbc:h2:file:./data/db
 
```

Comment:
Why keep it in file? I don't think this file will automatically be cleaned on shutting down server

Reply:
It will because of `spring.jpa.hibernate.ddl-auto = create-drop` in `application.properties.`

Reply:
Ok